### PR TITLE
feat(route): 首席经济学家

### PR DESCRIPTION
**Pull request by FledgeXu**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/chinacef
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by TonyRL** [comment]
Please follow v2 standard

**Comment by FledgeXu** [comment]
@TonyRL Already moved to V2 standard.

File: docs/finance.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ pageClass: routes
 
 # 金融
 
+### 首席经济学家论坛
```
Comment (TonyRL) [inline docs/finance.md:7]:
```suggestion
## 首席经济学家论坛
```
This should be H2 heading.

A H3 heading is also needed.

File: lib/v2/chinacef/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const utils = require('./utils');
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const parseDate = require('@/utils/dateParser');
```
Comment (TonyRL) [inline lib/v2/chinacef/index.js:4]: Please use `parseDate` from `@/utils/dateParser`

File: lib/v2/chinacef/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    });
+    const $ = cheerio.load(response.data);
+    const articlesLinkListNode = $('a[target=_black]')
+        .filter((_, item) => item.attribs.href.match(regex))
+        .map((_, item) => item.attribs.href);
+    const articlesLinkList = Array.from(new Set(articlesLinkListNode));
+    const out = await Promise.all(
+        articlesLinkList.map(async (item) => {
+            const articlesLink = rootUrl + item;
+            const cache = await ctx.cache.get(articlesLink);
```
Comment (TonyRL) [inline lib/v2/chinacef/index.js:23]: Please use `ctx.cache.tryGet(key, value)` instead

File: lib/v2/chinacef/index.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            const description = $('div[class=newsmaintext]').html();
+            const element = {
+                title,
+                author,
+                pubDate,
+                description: utils.renderDesc(description),
+                link: articlesLink,
+            };
+            ctx.cache.set(articlesLink, JSON.stringify(element));
+            return Promise.resolve(element);
```
Comment (TonyRL) [inline lib/v2/chinacef/index.js:59]:
```suggestion
            return element;
```

File: lib/v2/chinacef/index.js (hunk 2 above)
Comment (FledgeXu) [reply lib/v2/chinacef/index.js:4]: I don't get here. I already used the `parseDate` from `@/utils/dateParser`.

Comment (TonyRL) [reply lib/v2/chinacef/index.js:4]: My mistake. It should be `@/utils/parse-date`.

**Comment by FledgeXu** [comment]
@TonyRL Fixed the problems you mentioned.

File: lib/v2/chinacef/index.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    });
+    const $ = cheerio.load(response.data);
+
+    const articlesLinkListNode = $('a[target=_black]')
+        .filter((_, item) => item.attribs.href.match(regex))
+        .map((_, item) => item.attribs.href);
+    const articlesLinkList = Array.from(new Set(articlesLinkListNode));
+
+    const out = await Promise.all(
+        articlesLinkList.map(async (item) => {
```
Comment (TonyRL) [inline lib/v2/chinacef/index.js:23]:
```suggestion
        .map((_, item) => item.attribs.href);
        .toArray();

    const out = await Promise.all(
        articlesLinkListNode.map(async (item) => {
```

Comment (FledgeXu) [reply lib/v2/chinacef/index.js:23]: I need to remove the redundant links, I can't use `toArray()` here.

**Pull request merged**